tools/venvsetup.py

```
@@ -0,0 +1,205 @@
+    async def setup_project(self, project_path: Path, packages: List[str]) -> dict:
+        """Sets up a new Python project"""
+        project_path.mkdir(parents=True, exist_ok=True)
+        os.chdir(project_path)
+
+        # Setup virtual environment
+        ic("Creating virtual environment...")
+        self.run_command("uv venv")
+        try:
+            self.run_command("uv init")
+        except:
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)